x-pack/plugins/security/public/authentication/login/login_page.tsx

```diff
@@ -36,18 +36,33 @@ interface State {
   loginState: LoginState | null;
 }
 
-const infoMessageMap = new Map([
+const messageMap = new Map([
   [
     'SESSION_EXPIRED',
-    i18n.translate('xpack.security.login.sessionExpiredDescription', {
-      defaultMessage: 'Your session has timed out. Please log in again.',
-    }),
+    {
+      type: LoginFormMessageType.Info,
+      content: i18n.translate('xpack.security.login.sessionExpiredDescription', {
+        defaultMessage: 'Your session has timed out. Please log in again.',
+      }),
+    },
   ],
   [
     'LOGGED_OUT',
-    i18n.translate('xpack.security.login.loggedOutDescription', {
-      defaultMessage: 'You have logged out of Elastic.',
-    }),
+    {
+      type: LoginFormMessageType.Info,
+      content: i18n.translate('xpack.security.login.loggedOutDescription', {
+        defaultMessage: 'You have logged out of Elastic.',
+      }),
+    },
+  ],
+  [
```

Comment:
__note:__ Core's `onPreResponse` doesn't allow us to access response and extract original error message yet, so for now the best we can have is some generic error message and full error in the logs. There is a chance we can enhance Core to give us that info, but I didn't want to block on that.